docker/compose/docker-compose_hadoop284_hive233_spark240.yml

```diff
@@ -117,7 +117,7 @@ services:
       - ${HUDI_WS}:/var/hoodie/ws
 
   sparkmaster:
-    image: varadarb/hudi-hadoop_2.8.4-hive_2.3.3-sparkmaster_2.3.1:latest
+    image: varadarb/hudi-hadoop_2.8.4-hive_2.3.3-sparkmaster_2.4.0:latest
```

Comment:
@thesuperzapper @SemanticBeeng : We would need to create and push the docker image with the new name. Otherwise, you will notice the integration build failure. This is a chicken and egg problem.  Let me see what we can do about it. This should not block your progress (atleast till the PR is fully tested and ready to land)